README.md

```diff
@@ -107,3 +108,7 @@ $ sudo -E PATH=$PATH make AGENT_INIT=yes initrd-only
 
 For further details,
 see [the initrd builder documentation](initrd-builder/README.md).
+
+## Platform-Distro Compatibility Matrix
+
+![Compatibility Matrix](images/platform-distro-compatibility-matrix.png)
```

Comment:
Ah, I think @jodh-intel 's idea was to add the table in markdown, not as a binary image. Here is the raw md extracted from the other issue at: https://github.com/kata-containers/documentation/pull/186 :
```
 | | Alpine | CentOS | ClearLinux | EulerOS | Fedora | 
  |--|--|--|--|--|--|
  | **ARM64** | :heavy_check_mark: | :heavy_check_mark: |  | :heavy_check_mark: | :heavy_check_mark: | 
  | **PPC64le** | :heavy_check_mark: | :heavy_check_mark: |  |  | :heavy_check_mark: | 
  | **x86_64** | :heavy_check_mark: |:heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: |
```
which renders as

 | | Alpine | CentOS | ClearLinux | EulerOS | Fedora | 
  |--|--|--|--|--|--|
  | **ARM64** | :heavy_check_mark: | :heavy_check_mark: |  | :heavy_check_mark: | :heavy_check_mark: | 
  | **PPC64le** | :heavy_check_mark: | :heavy_check_mark: |  |  | :heavy_check_mark: | 
  | **x86_64** | :heavy_check_mark: |:heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: |